src/fundus/scraping/crawler.py

```diff
@@ -204,7 +204,10 @@ def build_extraction_filter() -> Optional[ExtractionFilter]:
                         f"is(are) not supported by {publisher.publisher_name}. Skipping publisher"
                     )
                 else:
-                    fitting_publishers.append(publisher)
+                    if not publisher.deprecated:
+                        fitting_publishers.append(publisher)
+                    else:
+                        logger.warning(f"Skipping deprecated publisher: {publisher.publisher_name}")
```

Comment:
I would either introduce a parameter to include deprecated publisher, since it is still possible to crawl them depending on IP address and User Agent, or don't skip them during crawling.

Reply:
Good idea, I added that.